jni/include/com_amazon_opendistroforelasticsearch_knn_index_nmslib_KNNIndex.h

```diff
@@ -1,50 +1,50 @@
+/* Header for class com_amazon_opendistroforelasticsearch_knn_index_nmslib_KNNIndex */
 
-#ifndef _Included_com_amazon_opendistroforelasticsearch_knn_index_v206_KNNIndex
-#define _Included_com_amazon_opendistroforelasticsearch_knn_index_v206_KNNIndex
+#ifndef _Included_com_amazon_opendistroforelasticsearch_knn_index_nmslib_KNNIndex
+#define _Included_com_amazon_opendistroforelasticsearch_knn_index_nmslib_KNNIndex
 #ifdef __cplusplus
 extern "C" {
 #endif
 /*
- * Class:     com_amazon_opendistroforelasticsearch_knn_index_v206_KNNIndex
+ * Class:     com_amazon_opendistroforelasticsearch_knn_index_nmslib_KNNIndex
  * Method:    saveIndex
  * Signature: ([I[[FLjava/lang/String;[Ljava/lang/String;Ljava/lang/String;)V
  */
-JNIEXPORT void JNICALL Java_com_amazon_opendistroforelasticsearch_knn_index_v206_KNNIndex_saveIndex
+JNIEXPORT void JNICALL Java_com_amazon_opendistroforelasticsearch_knn_index_nmslib_KNNIndex_saveIndex
   (JNIEnv *, jclass, jintArray, jobjectArray, jstring, jobjectArray, jstring);
 
 /*
- * Class:     com_amazon_opendistroforelasticsearch_knn_index_v206_KNNIndex
+ * Class:     com_amazon_opendistroforelasticsearch_knn_index_nmslib_KNNIndex
  * Method:    queryIndex
  * Signature: (J[FI)[Lcom/amazon/opendistroforelasticsearch/knn/index/KNNQueryResult;
  */
-JNIEXPORT jobjectArray JNICALL Java_com_amazon_opendistroforelasticsearch_knn_index_v206_KNNIndex_queryIndex
+JNIEXPORT jobjectArray JNICALL Java_com_amazon_opendistroforelasticsearch_knn_index_nmslib_KNNIndex_queryIndex
   (JNIEnv *, jclass, jlong, jfloatArray, jint);
 
 /*
- * Class:     com_amazon_opendistroforelasticsearch_knn_index_v206_KNNIndex
+ * Class:     com_amazon_opendistroforelasticsearch_knn_index_nmslib_KNNIndex
  * Method:    init
  * Signature: (Ljava/lang/String;[Ljava/lang/String;Ljava/lang/String;)J
  */
-JNIEXPORT jlong JNICALL Java_com_amazon_opendistroforelasticsearch_knn_index_v206_KNNIndex_init
+JNIEXPORT jlong JNICALL Java_com_amazon_opendistroforelasticsearch_knn_index_nmslib_KNNIndex_init
   (JNIEnv *, jclass, jstring, jobjectArray, jstring);
 
 /*
- * Class:     com_amazon_opendistroforelasticsearch_knn_index_v206_KNNIndex
+ * Class:     com_amazon_opendistroforelasticsearch_knn_index_nmslib_KNNIndex
  * Method:    gc
  * Signature: (J)V
  */
-JNIEXPORT void JNICALL Java_com_amazon_opendistroforelasticsearch_knn_index_v206_KNNIndex_gc
+JNIEXPORT void JNICALL Java_com_amazon_opendistroforelasticsearch_knn_index_nmslib_KNNIndex_gc
   (JNIEnv *, jclass, jlong);
 
 /*
- * Class:     com_amazon_opendistroforelasticsearch_knn_index_v206_KNNIndex
+ * Class:     com_amazon_opendistroforelasticsearch_knn_index_nmslib_KNNIndex
  * Method:    initLibrary
  * Signature: ()V
  */
-JNIEXPORT void JNICALL Java_com_amazon_opendistroforelasticsearch_knn_index_v206_KNNIndex_initLibrary
+JNIEXPORT void JNICALL Java_com_amazon_opendistroforelasticsearch_knn_index_nmslib_KNNIndex_initLibrary
```

Comment:
Lets keep the versioning in the interface so that it would be easy to support backward compatibility when we find one.  lets add v208 similar to v206

Reply:
Makes sense. Will update.